package.json

```diff
@@ -25,10 +25,6 @@
   ],
   "author": "ThinkingLabs",
   "license": "MIT",
-  "engineStrict": true,
-  "engines": {
-    "node": "^16"
```

Comment:
My reading of https://github.com/thinkinglabs/aws-iam-policy/commit/6b7279df20bb5f77817e0500fe20ef51c5028987 is that we pinned to node 16 to get CI to run on that version without needing to specify it in multiple places. But the GitHub node action already supports parsing `volta` from package.json. So we don't need a strict `engines` version here. https://github.com/actions/setup-node/blob/aca7b64a59c0063db8564e0ffdadd3887f1cbae5/src/util.ts#L21-L27

Reply:
True.
Good catch about support for `volta` in `actions/setup-node`.

That makes me think I should set up a matrix build to run the tests on several Node.js versions.